### PR TITLE
MES-5006 - Vehicle Checks Results not Calculating Correctly for Cat C

### PR DESCRIPTION
**Pull request by TomBillingtonUK**
# Description and relevant Jira numbers
- Fixes issue with serious faults not being correctly recorded in RSIS for CE and C1E tests

## Pull Request checklist

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop

## Sign off process checklist

- [ ] Code has been tested manually
- [ ] Tested by QA
- [ ] PO's approval
- [x] Reviewers selected in Github
- [ ] PR link added to JIRA

### CODE REVIEW

**Review by szabikr — APPROVED** [review]
looks good

File: src/functions/uploadToRSIS/application/data-mapper/cat-ce-mapper.ts [hunk 1]
```diff
@@ -9,14 +8,22 @@ import {
...
 
-  const m: DataField[] = [...catCDataFields,
+  const m: DataField[] = [
+    ...catCDataFields,
+    field('VEHICLE_CHECKS_COMPLETED', formatQuestionCompleted(t, 2)),
+    field('VEHICLE_CHECKS_TOTAL', formatQuestionFaultCE(t)),
+    field('VEHICLE_CHECKS_SERIOUS', formatQuestionSeriousCE(t)),
     field('UNCOUPLE_RECOUPLE_TOTAL', formatManoeuvreFault(t, 'uncoupleRecouple.fault')),
     field('UNCOUPLE_RECOUPLE_SERIOUS', formatManoeuvreSerious(t, 'uncoupleRecouple.fault')),
     field('UNCOUPLE_RECOUPLE_DANGEROUS', formatManoeuvreDangerous(t, 'uncoupleRecouple.fault')),
```
Comment (rossfauldsbjss) [inline src/functions/uploadToRSIS/application/data-mapper/cat-ce-mapper.ts:29]: UNCOUPLE_RECOUPLE_COMMENT - is this fixed separately to this?

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/functions/uploadToRSIS/application/data-mapper/cat-ce-mapper.ts (hunk 1 above)
Comment (TomBillingtonUK) [reply src/functions/uploadToRSIS/application/data-mapper/cat-ce-mapper.ts:29]: Separate ticket which is already in this branch :)

**Pull request merged**